lightly/api_client/api_workflow_upload_embeddings.py

```diff
@@ -0,0 +1,64 @@
+from __future__ import annotations
+from typing import TYPE_CHECKING
+if TYPE_CHECKING:
+    from lightly.api_client.api_workflow_client import ApiWorkflowClient
+
+import csv
+from typing import List
+
+from lightly.api.upload import upload_file_with_signed_url
+from lightly.openapi_generated.swagger_client.models.dataset_embedding_data import DatasetEmbeddingData
+from lightly.openapi_generated.swagger_client.models.write_csv_url_data import WriteCSVUrlData
+
+
+class UploadEmbeddingsMixin:
+    def upload_embeddings(self: ApiWorkflowClient, path_to_embeddings_csv: str, name: str = None):
```

Comment:
Docstring for class and method are missing

Reply:
The docstring should include the information on how this function works: 
1. Checks if embedding exists
2. Changes current embedding_id to whatever
3. ....

Reply:
Does both the class and method need to be documented by a doctoring? The class is never meant to be implemented, it is a Mixin, thus I would only document the method.